### PR TITLE
feat(packages): add inline Alert component

### PR DESCRIPTION
**Pull request by BalbinaK**
## Jira
Fixes [FRON-593](https://finn-jira.atlassian.net/browse/FRON-593)

## Description
Add inline alert component to feedback indicators based on Figma sketches:
https://www.figma.com/file/gpHv46zswMNeepUDyQfpS8/Feedback-indicators?node-id=274%3A31588

Screenshot 2022-05-30 at 15 28 07

Screenshot 2022-05-30 at 15 29 03

### CODE REVIEW

File: packages/alert/src/component.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
+import React, { PropsWithChildren, ReactElement } from 'react';
+import { AlertProps } from '.';
+
+export function Alert({ type, children }: PropsWithChildren<AlertProps>) {
+  const { color, icon } = styleMap[type];
```
Comment (digitalsadhu) [inline packages/alert/src/component.tsx:5]: this is nice 👍

**Review by digitalsadhu — APPROVED** [review]
This looks great!
I appreciate the screen grabs in the PR as well.

File: packages/alert/src/component.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+  return (
+    <div
```
Comment (pearofducks) [inline packages/alert/src/component.tsx:8]: Does this inherit classes passed onto it?

e.g. if I do `<Alert type="info" className="my-amazing-override">` will it put the `my-amazing-override` class on the root `div`?

Comment (BalbinaK) [reply packages/alert/src/component.tsx:13]: Good point! I handled inheritance of `className` and inline `style` values in [e4e289f](https://github.com/fabric-ds/react/pull/106/commits/e4e289f71b4bd9ea2efce868b2336df36fc4fdce)

**Review by pearofducks — APPROVED** [review]

**Pull request merged**